packages/rmb_direct_client/lib/sign.ts

```diff
@@ -16,10 +15,7 @@ export function sign(payload: string | Uint8Array, signer: KeyringPair) {
   return sigPrefixed;
 }
 
-export async function createShared(pubKey: Uint8Array, mnemonic: string) {
-  const seed = await bip39.mnemonicToSeed(mnemonic);
-  const privateKey = new Uint8Array(seed).slice(0, 32);
-  const pointX = secp.getSharedSecret(privateKey, pubKey);
-  const key = await secp.utils.sha256(pointX.slice(1, 33));
-  return key;
+export function createShared(pubKey: Uint8Array, hexSeed: string) {
```

Comment:
any user can use this client without using the grid client, we shouldn't make work on hex seed only. please handle the both cases